net/tor/Config.in

```diff
@@ -0,0 +1,10 @@
+if PACKAGE_tor
+
+config TOR_RELAY
+	bool "Enable bridge/relay support"
+	help
+		If only client (entry node) functionality is required, disabling
+		this feature shrinks the tor executable by around 400 kiB.
+	default y
```

Comment:
I'm not sure if this should be the default option. I know few people which run tor bridge/relay on the router and for them it would require to recompile tor package. 
I could image to create another TOR package VERSION where this option is enabled.

Reply:
I think that a similar approach implemented in FFmpeg will be more appropriate https://github.com/openwrt/packages/blob/master/multimedia/ffmpeg/Makefile#L315

Reply:
> I'm not sure if this should be the default option. I know few people which run tor bridge/relay on the router and for them it would require to recompile tor package.

I also believe most people (like me) just want to run an entry node. My approach was adding such possibility without regressing existing functionality

> I could image to create another TOR package VERSION where this option is enabled.

I would also prefer having two separate packages. Something like `tor` (complete, with the current name) and `tor-basic` (entry node only), perhaps?

As an aside, I have a potentially more controversial patch which I didn't send as part of this series, which makes the tor build rely on the global hardening settings of the OpenWrt build. If hardening is disabled, the tor executable size is significantly reduced.

> I think that a similar approach implemented in FFmpeg will be more appropriate https://github.com/openwrt/packages/blob/master/multimedia/ffmpeg/Makefile#L315

Great, I'll take a look, thanks!